### PR TITLE
optimum 1.18.1 

### PR DESCRIPTION
**Pull request by msentissi**
optimum 1.18.1 

**Destination channel:** Defaults

### Links

- [PKG-6356]
- dev_url:        https://github.com/huggingface/optimum/tree/v1.18.1
- conda_forge:    https://github.com/conda-forge/optimum-feedstock
- pypi:           https://pypi.org/project/optimum/1.18.1
- pypi inspector: https://inspector.pypi.io/project/optimum/1.18.1

### Explanation of changes:

- updated version and dependencies
- updated `about` section
- added `entry_point`

[PKG-6356]: https://anaconda.atlassian.net/browse/PKG-6356?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,16 +8,18 @@ package:
...
-  sha256: a74e051c4d776a900b6452a12a36e0afadbebee112a8205a30adac9216a4991b
+  sha256: c962cc02d55d698461cbc2e3fcf5522fab83ed846c3f904f5bed3efdc48a702b
 
 build:
   number: 0
   # transformers >=4.26.0 is not available on s390x
-  skip: true  # [py<37 or s390x]
   # Dropping ppc because of various pytorch build issues
-  skip: True  # [py>310 and (linux and ppc64le)]
+  skip: true  # [linux and (ppc64le or s390x)]
```
Comment (SC426) [inline recipe/meta.yaml]: No need to worry about ppc64le since we don't build for it

Comment (SC426) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [s390x]
```

**Review by ifitchet — APPROVED** [review]
of interest, conda-forge add `optimum-cli --help` as a test.

**Comment by msentissi** [comment]
@ifitchet @SC426 Thank you! I've removed the superfluous skip statement and added the cli test.

**Review by SC426 — APPROVED** [review]

**Pull request merged**